Makefile

```diff
@@ -62,6 +62,10 @@ docker/build/apk/shell:
 		--name apkbuild \
 		--rm \
 		-it \
+		-e AWS_SECRET_ACCESS_KEY \
+		-e AWS_ACCESS_KEY_ID \
+		-e AWS_SESSION_TOKEN \
```

Comment:
Where are you getting a SESSION_TOKEN from?

Reply:
This is set by `aws-vault exec` when assuming a role. These are the canonical AWS envs for the purpose of sessions. In docker, setting `docker run -e FOO` will pass through whatever the current value is of `FOO` in your current shell. In otherwords, doing `-e FOO=${FOO}` is identical to `-e FOO`